src/SDKs/Azure.Base/data-plane/Azure.Base/Http/HttpMessage.cs

```diff
@@ -23,8 +23,8 @@ public abstract partial class HttpMessage  : IDisposable
 
         public abstract void AddHeader(HttpHeader header);
 
-        public virtual void AddHeader(string name, string value)
-            => AddHeader(new HttpHeader(name, value));
+        public virtual void AddHeader(string name, HeaderValues values)
```

Comment:
What's the reason to have HeaderValues as input?

Reply:
For headers that are sent multiple times with different values.

```
Header: value1
Header: value2
```

Reply:
Currently you can just call AddHeader multiple times with the same name.